{"uuid": "79461c9d-2405-4710-b6b3-0e71b75a7068", "children": ["13e1ff07-9260-4642-8ffc-f06206057098", "fee9d88a-00da-4520-9db0-669c2833c164", "8a302d3f-1d84-41d1-9a3e-95cac1a4d057", "d95f9773-966d-41c9-ad5a-b41105cbe71a", "cbcfeeb4-64a6-46dd-bf98-61e5af87bb2b", "1b834c78-2da9-4dfd-bbbe-792d89d79be9"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408535, "stop": 1699988408535}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408545, "stop": 1699988408545}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408551, "stop": 1699988408551}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408557, "stop": 1699988408557}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408563, "stop": 1699988408563}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408569, "stop": 1699988408569}], "start": 1699988408569, "stop": 1699988408573}